.circleci/config.yml

```diff
@@ -55,10 +84,44 @@ jobs:
           PROJECT="circleci/aws-code-deploy-orb-test/build"
           curl -X POST $BASE/$PROJECT?circle-token=$CIRCLECI_API_TOKEN
 
-  release:
+  slack-notify-hold:
+    machine: true
+    steps:
+      - slack/notify:
+          message: "Orbs from commit ${CIRCLE_SHA1} are ready for review/approval: https://circleci.com/workflow-run/${CIRCLE_WORKFLOW_ID}"
+          mentions: rose,eddiewebb
+
+  dev-promote-patch:
     executor: cli
     steps:
       - checkout
+
+      - slack/notify:
+          message: "Orb publishing was approved for any orbs modified in commit ${CIRCLE_SHA1}: https://circleci.com/workflow-run/${CIRCLE_WORKFLOW_ID}"
+          mentions: rose,eddiewebb
+
+
+      - run:
+          name: Publish any modified orbs
+          shell: /bin/bash -exo pipefail
+          command: |
+            COMMIT_RANGE=$(echo $CIRCLE_COMPARE_URL | sed 's:^.*/compare/::g')
```

Comment:
@iynere I've heard rumor that CIRCLE_COMPARE_URL doesn't work with build processing, but I presume this is working fine, right?

Reply:
we can't test w/thecurrent code b/c it depends on being on master, but i'm going to modify the dev-release script so it also only publishes dev releases if an orb's source code changes @ndintenfass